GitCommands/Git/GitCommandHelpers.cs

```diff
@@ -993,7 +993,7 @@ private static GitItemStatus GitItemStatusFromStatusCharacter(StagedStatus stage
 
             return new GitItemStatus
             {
-                Name = fileName.Trim(),
+                Name = fileName,
```

Comment:
Any thoughts as to why we had `Trim()` here?

Reply:
It has been like this for a long time, so I guess it was just an assumption how things work. Maybe adjusted to tests?

This may not be complete, to handle copied files the rename in GitItemStatusFromCopyRename() must be removed too (affects git-diff and git-status porecelain v1).
This is a small part though.

Reply:
Removed Trim() for rename, copy, added test case